translations/create_translation_json.py

```diff
@@ -0,0 +1,126 @@
+        if param_name:
+            key = create_key(function_name=name, parameter=parameter.arg_name, version=version)
+            output_dict[key] = param_name
+        if description:
+            description_key = create_key(
+                function_name=name, parameter=parameter.arg_name, description=True, version=version
+            )
+            output_dict[description_key] = _convert_to_rendering_format(description)
+
+
+# Generation of the keys and values
+def _generate_key_for_function(function: typing.Callable, name: str, output_dict: dict, version: Optional[str] = None):
+    docstring = str(function.__doc__) if function.__doc__ else ""
+    parsed_docstring = docstring_parser.parse(docstring, docstring_parser.DocstringStyle.GOOGLE)
+
+    # Function name
+    short_description = parsed_docstring.short_description or ""
+    output_dict[create_key(function_name=name, version=version)] = _parse_docstring_element_text(short_description)[0]
+
+    # Function description
+    if parsed_docstring.long_description:
+        long_desc_key = create_key(function_name=name, description=True, version=version)
+        output_dict[long_desc_key] = _convert_to_rendering_format(parsed_docstring.long_description)
+
+    # Function parameters
+    if parsed_docstring.params:
+        _generate_key_for_parameters(name, output_dict, parsed_docstring.params, version)
+
+    # Function return
+    if parsed_docstring.returns:
+        return_name = _parse_docstring_element_text(parsed_docstring.returns.description)[0]
+        if return_name:
+            return_key = create_key(function_name=name, output=True, version=version)
+            output_dict[return_key] = return_name
+
+
+# Add versioning if the function is versioned
+def _generate_translation_mapping_for_functions(output_dict: dict, module):
+    functions_to_export = getattr(module, COGNITE, [])
+    functions_map = inspect.getmembers(module, inspect.isfunction)
+
+    for name, function in filter(lambda f: f[0] in functions_to_export, functions_map):
+        if versioning.is_versioned(function):
+            _generate_key_for_versioned_function(function, output_dict)
+        else:
+            _generate_key_for_function(function, name, output_dict=output_dict)
+
+
+# Process versioned function by adding only the deprecated version to the keys.
+# The latest version does not have a version number in the key.
+def _generate_key_for_versioned_function(function, output_dict):
+    function_name = versioning.get_name(function)
+    versions = versioning.get_versions(function_name)
+
+    for version in versions:
+        func = versioning.get(function_name, version)
+        _generate_key_for_function(
+            func,
+            function_name,
+            output_dict=output_dict,
+            version=version if version != versions[-1] else None,
+        )
+
+
+# Write the keys and values to the JSON file
+def create_mapping_for_translations(module):
```

Comment:
module is not used as here you are going through all the indsl functions. 
You can split this into two: 
def create_mapping_for_indsl_translations(): 
that spesifies module=indsl, writes to file and calls
def create_mapping_for_translations(module) that executes the for loop

Reply:
the output file should also be saved to translations/en folder